source/test/F0.Minesweeper.Logic.Tests/MinefieldDifficultyTest.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace F0.Minesweeper.Logic.Tests
 {
-	public class MinefieldOptionsTest
+	public class MinefieldDifficultyTest
```

Comment:
please add tests for the associated "GenerationOption" to each difficulty level

Reply:
You mean something like this?
```
MinefieldOptions.Easy.GenerationOption.Should().Be(MinefieldFirstUncoverBehavior.WithoutAdjacentMines);
MinefieldOptions.Medium.GenerationOption.Should().Be(MinefieldFirstUncoverBehavior.CannotYieldMine);
[...]
```